simpleclient/src/main/java/io/prometheus/client/SimpleCollector.java

```diff
@@ -107,6 +109,17 @@ protected void initializeNoLabelsChild() {
   }
 
   /**
+   * Return label names and values as map.
+   */
+  protected Map<String, String> labelsMap(List<String> childLabels) {
+    Map<String, String> labels = new HashMap<String, String>();
```

Comment:
Of course, but I guess we've saved much more Lists here and there in the removed code. :-)